modules/displayBooks.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable*/
```

Comment:
Please don't disable linters. You can use `npx eslint . --fix` command to fix the linter error. 
There are a few reasons for using linters:
Catching syntax errors is more efficient. There is no need to debug simple mistakes like typos - the linter does it for you.
The entire codebase looks like written by one person.
Programmers can focus on solving problems, instead of cleaning up the code.

You can search for the linter error on their [documentation page](https://eslint.org/docs/latest/) and solve it.